crates/node/ops/worker_threads.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::generic_error;
+use deno_core::error::AnyError;
+use deno_core::op2;
+use deno_core::url::Url;
+use deno_core::OpState;
+use deno_fs::FileSystemRc;
+use std::path::Path;
+use std::path::PathBuf;
+use std::rc::Rc;
+
+use crate::resolution;
+use crate::NodePermissions;
+use crate::NodeResolver;
+use crate::NpmResolverRc;
+
+fn ensure_read_permission<P>(state: &mut OpState, file_path: &Path) -> Result<(), AnyError>
+where
+    P: NodePermissions + 'static,
+{
+    let resolver = state.borrow::<NpmResolverRc>();
+    let permissions = state.borrow::<P>();
+    resolver.ensure_read_permission(permissions, file_path)
+}
+
+#[op2]
+#[string]
+pub fn op_worker_threads_filename<P>(
```

Comment:
Looks like worker threads are disabled. Do we need to add this op?

Reply:
We could probably remove this file